frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/TraverseStepConfirmationDialog.java

```diff
@@ -21,14 +22,19 @@ public enum Result {
 
   @UIEffect
   public Result show(Project project) {
-    int result = MessageDialogBuilder.yesNoCancel(title, message)
-        .cancelText(getNonHtmlString("action.GitMachete.BaseTraverseAction.dialog.cancel-traverse"))
-        .show(project);
+    val yesText = getNonHtmlString("action.GitMachete.BaseTraverseAction.dialog.yes");
+    val yesAndQuitText = getNonHtmlString("action.GitMachete.BaseTraverseAction.dialog.yes-and-quit");
```

Comment:
Still no actual handling for that case (just behaves as plain `Quit` for now)

Reply:
fine 👍 

- [ ] **C H E C K B O X**